lib/tasks/cortex/core/media.rake

```diff
@@ -29,15 +29,17 @@ namespace :cortex do
                            },
                          metadata:
                            {
-                             styles: {
-                               large: {geometry: '1800x1800>', format: :jpg},
-                               medium: {geometry: '800x800>', format: :jpg},
-                               default: {geometry: '300x300>', format: :jpg},
-                               mini: {geometry: '100x100>', format: :jpg},
-                               micro: {geometry: '50x50>', format: :jpg},
-                               post_tile: {geometry: '1140x', format: :jpg}
+                            image_styles: {
```

Comment:
Ok I originally had that in a past commit, but I was told to change it. So what I do instead is I just change `image_styles` to `styles` if it's an image asset